dts/bindings/haptics/ti,drv2605.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2024 Cirrus Logic, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  DRV2605 Haptic Driver for ERM and LRA with built-in library and smart-loop
+  architecture.
+
+compatible: "ti,drv2605"
+
+include: i2c-device.yaml
+
+properties:
+  ti,actuator-mode:
```

Comment:
We typically only use dashes in dts properties, and exclude the vendor prefix, so `ti,actuator-mode` would be `actuator-mode`

Reply:
ack